src/cljs/main/broadfcui/page/workspace/monitor/workflow_details.cljs

```diff
@@ -102,6 +102,21 @@
            (create-field name (display-value value)))
          log-map)])))
 
+(defn- failure-tree [failures]
+  [comps/Tree {:start-collapsed? false
+               :label "Failures"
```

Comment:
Since we want to avoid using +/- for collapsing the tree, and instead use Show/Hide to be consistent with the rest of the page, we should not pass comps/Tree a label, and instead have create the label outside the tree